planning/autoware_planning_validator/include/autoware/planning_validator/utils.hpp

```diff
@@ -41,8 +41,14 @@ void calcSteeringAngles(
 
 std::pair<double, size_t> calcMaxCurvature(const Trajectory & trajectory);
 
+void calc_interval_distance(
+  const Trajectory & trajectory, std::vector<double> & interval_distance_arr);
+
```

Comment:
`arr` means array?

Reply:
https://github.com/tier4/autoware_universe/blob/fb7f3752ef9ad2fb92ce80d8b51267fb8cb99162/planning/autoware_planning_validator/src/utils.cpp#L238-L239
better to use https://github.com/tier4/autoware_universe/blob/fb7f3752ef9ad2fb92ce80d8b51267fb8cb99162/planning/autoware_planning_validator/src/utils.cpp#L238-L239

Reply:
or _vectory

Reply:
change variable name from _arr to _vector in the https://github.com/autowarefoundation/autoware_universe/pull/10385/commits/3e028c69c7d5057111f64e4045ae28367c1e944b